I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 21, in script
    show dice at halfsize:
  File "game/script.rpy", line 21, in <module>
    show dice at halfsize:
NameError: name 'halfsize' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 21, in script
    show dice at halfsize:
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\ast.py", line 1428, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\ast.py", line 1387, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\ast.py", line 1387, in <listcomp>
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 21, in <module>
    show dice at halfsize:
NameError: name 'halfsize' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.1.23060707
Emi VN 1.0
Mon Oct  7 21:15:55 2024
